docs/proposals/parameter-distribution.md

```diff
@@ -0,0 +1,85 @@
+# Proposal for Supporting various parameter distributions in Katib
+
+## Summary
+The goal of this project is to enhance the existing Katib Experiment APIs to support various parameter distributions such as uniform, log-uniform, and qlog-uniform. Then extend the suggestion services to be able to configure distributions for search space using libraries provided in each framework.
```

Comment:
It would be nice to identify goals and no goals for this project.

Reply:
Added.